app/src/main/java/com/antosik/benchproject/app/movie/details/viewModel/MovieDetailsViewModel.kt

```diff
@@ -24,10 +24,13 @@ class MovieDetailsViewModel @Inject constructor(
         get() = _isLoaderVisible
     val isScreenElementsVisible: LiveData<Boolean>
         get() = _isVisibleScreenElements
+    val isEmptyDatabase: LiveData<Boolean>
```

Comment:
You rather should not use names like `Database` on UI layer (view, ViewModel) 😄. Use a more abstract name.

Reply:
In this case, you probably should follow what you did in XML, so it should be just `isPlaceholderVisible`. You should try to name UI layer variables more "what it changes on the screen when set to true?" than "why it is set to true?".

Reply:
I changed it 😄